### PR TITLE
#164797136 Configure Authors Haven to use Postgres

### PR DESCRIPTION
**Pull request by NedyUdombat**
#### What does this PR do?
Configure Authors haven to use postgres as its database engine

#### Description of Task to be completed?
* uninstall mongo and it's dependencies
* install postgres and its dependencies (pg, pg-hstore)
* install babel and its dependencies
* setup babel configuration file
* install sequelize and its dependencies
* setup sequelize configuration file
* initialise Authors Haven to use sequelize
* create a database connnection point to sequelize (`server/config/config.js`)
* create a template environment file to store database connection details.
* remove all old models and confgurations that were based on mongo database.

#### How should this be manually tested?
* clone `https://github.com/andela/zinnia-ah-backend.git`
* checkout to  a new branch = `ch-setup-postgres-164797136`
* pull from the remote branch to update the local branch
* run `npm install`
* run `sudo npm install -g pg`
* run `cp .env.example .env`
* create a database
* fill in the details in your `.env`
* run `sequelize db:migrate`

#### Any background context you want to provide?
None 

#### What are the relevant pivotal tracker stories?
#164797136 

#### Screenshots (if appropriate)
None

#### Questions:
None.

### CODE REVIEW

**Review by emmsdan — APPROVED** [review]

**Review by emmsdan — DISMISSED** [review]
you created a .babelrc file and installed babel dependencies but you have not configured the *scripts* to run it. i.e `npm run build`

**Comment by NedyUdombat** [comment]
> you created a .babelrc file and installed babel dependencies but you have not configured the _scripts_ to run it. i.e `npm run build`

This was to help compile sequelize to run, `npm run build` will be setup when deploying the base app to heroku

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["env"]
+  }
```
Comment (Japhethca) [inline .babelrc]: fix alignment

File: server/config/config.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+module.exports = {
```
Comment (Japhethca) [inline server/config/config.js:5]: let use es6 export

File: server/migrations/20190325172457-create-users.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+module.exports = {
```
Comment (Japhethca) [inline server/migrations/20190325172457-create-users.js:1]: same as above

File: server/models/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (Japhethca) [inline server/models/index.js:1]: same as above

**Review by Japhethca — APPROVED** [review]

**Pull request merged**